test/e2e/serial/tests/resource_hostlevel.go

```diff
@@ -371,6 +380,7 @@ var _ = Describe("[serial][hostlevel] numaresources host-level resources", Seria
 				corev1.PodQOSBurstable,
 			),
 			Entry("[test_id:74255][tier3][qos:be][unsched] with ephemeral storage, multi-container",
+				Label("tier2", "qos:be", "unsched"),
```

Comment:
should be tier3 inside the Label not tier2

Reply:
thanks, fixed